src/components/shared/TextAreaField/index.tsx

```diff
@@ -30,7 +30,7 @@ const TextAreaField = ({
   onBlur,
   value,
   size = 'md',
-  characterCounter = true,
+  characterCounter = false, // EASI-3456
```

Comment:
With this change we can probably remove the ~19 or so instances where we're explicitly passing `characterCounter={false}`